src/library_fs.js

```diff
@@ -904,7 +904,11 @@ FS.staticInit();
       if (!link.node_ops.readlink) {
         throw new FS.ErrnoError({{{ cDefs.EINVAL }}});
       }
-      return PATH_FS.resolve(FS.getPath(link.parent), link.node_ops.readlink(link));
+      const target = link.node_ops.readlink(link);
```

Comment:
We tend to avoid `let` and `const` in JS library code still.   Sadly using `var` everywhere lends to better minified code size.